### PR TITLE
Add updated and new help files

### PR DESCRIPTION
**Pull request by jamienicholls**

### CODE REVIEW

File: help/help_src/2. Business Reference Model.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
 # Business Reference Model
 
-Here is stuff about the Business Reference Model, with an example of including an image
+## Architecture Portal – Business Reference Model 
 
-<img src="img/screenshot-brm.png" />
```
Comment (blackwidow207) [inline help/help_src/2. Business Reference Model.md:5]: Is this screenshot still required?

Comment (jamienicholls) [reply help/help_src/2. Business Reference Model.md:5]: For now yup

Comment (blackwidow207) [reply help/help_src/2. Business Reference Model.md:5]: is it referenced elsewhere now? if so I may have missed it

Comment (jamienicholls) [reply help/help_src/2. Business Reference Model.md:5]: Ill remove it :)

Comment (blackwidow207) [reply help/help_src/2. Business Reference Model.md:5]: its referenced on the main readme file of this repo, maybe just leave it. I was only checking if the reference to the screenshot on this page was still needed

**Review by blackwidow207 — APPROVED** [review]

**Pull request merged**